tests/ckeditor/basepath.js

```diff
@@ -31,6 +32,20 @@ describe( 'basePath', function() {
 		window.CKEDITOR_BASEPATH = 'http://foo.com/ckeditor/';
 		expect( CKEDITOR._getBasePath() ).to.equal( 'http://foo.com/ckeditor/' );
 	} );
+
+	function testGetBasePathFromTag( url, expectedBasePath ) {
```

Comment:
Hum... we need to figure out something here, because this is a must have for us.